pkg/blobstore/configuration/new_blob_access.go

```diff
@@ -240,6 +241,7 @@ func newNestedBlobAccessBare(configuration *pb.BlobAccessConfiguration, creator
 					ReadTimeout:     readTimeout,
 					WriteTimeout:    writeTimeout,
 				})
+
```

Comment:
Nit: unnecessary whitespace change? Better revert this to prevent merge conflicts for others.